pages/common/git-root.md

```diff
@@ -0,0 +1,13 @@
+# git root
+
+> Print Current Working directory.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-root>.
+
+- Print directory currently opened in a Git repository, from root directory:
+
+`git root`
+
+- Print directory currently opened in a Git repository, from beginning of repository:
```

Comment:
```suggestion
- Print the current working directory, relative to the root of the current Git repository:
```

Reply:
This is a little man pagey

Reply:
I don't think that has to be a bad thing.

Reply:
Neither do I.  You know, "the" is used in man pages, so should we try to avoid it? 😄

@marchersimon I'd remove the comma in that suggestion though.

Reply:
Ok, after reading a few times I see your point, however I shouldn't of had to read it a few times to understand what it meant

Reply:
No...but not everyone does.  I, for one, knew what it mean with one glance. 🤷🏻‍♂️

Reply:
Well I obviously new what it does, since I wrote this :) However it just sounds a little confusing however if you agree with it then we will use it

Reply:
"relative to" is common and it's the most succinct way of putting it IMO; we can't make everything perfect.  I think it's fine.